apps/main/src/llamalend/features/user-position-history/cells/EventTypeCell.tsx

```diff
@@ -0,0 +1,52 @@
+import AddCircleIcon from '@mui/icons-material/AddCircleOutlined'
+import AutoAwesomeIcon from '@mui/icons-material/AutoAwesomeOutlined'
+import CancelIcon from '@mui/icons-material/CancelOutlined'
+import DownloadIcon from '@mui/icons-material/DownloadOutlined'
+import RemoveCircleIcon from '@mui/icons-material/RemoveCircleOutlined'
+import UploadIcon from '@mui/icons-material/UploadOutlined'
+import WarningAmberIcon from '@mui/icons-material/WarningAmberOutlined'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import type { CellContext } from '@tanstack/react-table'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import type { ParsedUserLendCollateralEvent, UserLendCollateralEventType } from '../hooks/useUserLendCollateralEvents'
+
+const { Spacing } = SizesAndSpaces
+
+const getIcon = (type: UserLendCollateralEventType) => {
```

Comment:
we should declare a `const icons: Record<UserLendCollateralEventType, ReactNode>` to make sure we always cover all cases